.travis.yml

```diff
@@ -74,6 +76,3 @@ deploy:
     local_dir: docs/public
     skip_cleanup: true
     github_token: ${GITHUB_TOKEN}
-    on:
```

Comment:
Was removing this an accident?

Reply:
Nope, I think the doc needs to be updated even between two versions.
If I don't do that, the doc changes made by this PR will not be published.